examples/tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can we inherit from https://github.com/immutable/imx-core-sdk/blob/main/tsconfig.json?

Reply:
No we want this to be separate from the core-sdk.

Reply:
I meant is it worth inheriting any of the `compilerOptions` for eg. If not needed that is ok